src/components/binding/binding-renderer.js

```diff
@@ -124,9 +128,11 @@ genotet.BindingRenderer.prototype.EXON_LABEL_SIZE = 6;
 genotet.BindingRenderer.prototype.BED_LABEL_SIZE = 13;
 /** @const {number} */
 genotet.BindingRenderer.prototype.BED_MIN_WIDTH = 3;
+/** @const {number} */
+genotet.BindingRenderer.prototype.BED_HEIGHT_PROPORTION = 0.8;
 
 /** @const {!Array<number>} */
-genotet.BindingRenderer.ZOOM_EXTENT = [1, 65536];
+genotet.BindingRenderer.ZOOM_EXTENT = [1, 15673503];
```

Comment:
I calculated it in this way: 
the range of overview: range = 197194699 - 2999997 = 194194702,
the width of canvas: width = 1239px,
the width of bed label (majority amount): label = 100px,
to guarantee that the labels of neighbor bed rects could be seen clearly, we need the max zoom extent: maxExtent = range * label / width = 15673503.